core-services/supplemental-ci-images/ubi-nodejs-10.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ImageStream
+metadata:
+  name: ubi-nodejs-10
+  namespace: ocp
+spec:
+  tags:
+  - name: "7"
+    from:
+      kind: DockerImage
+      name: registry.access.redhat.com/ubi7/nodejs-10:latest
+    importPolicy:
+      scheduled: true
+  - name: "8"
+    from:
+      kind: DockerImage
+      name: registry.access.redhat.com/ubi8/nodejs-10:latest
```

Comment:
is this even available here?  ubi8/rhel8 content is not supposed to be on registry.access.redhat.com, only registry.redhat.io (requires auth)